src/connection.cpp

```diff
@@ -148,13 +148,13 @@ namespace oxen::quic
         return 0;
     }
 
-    int recv_rx_key(ngtcp2_conn* conn, ngtcp2_crypto_level level, void* user_data)
+    int recv_rx_key(ngtcp2_conn* conn, ngtcp2_encryption_level level, void* user_data)
     {
         // fix this
         return 0;
     }
 
-    int recv_tx_key(ngtcp2_conn* conn, ngtcp2_crypto_level levl, void* user_data)
+    int recv_tx_key(ngtcp2_conn* conn, ngtcp2_encryption_level levl, void* user_data)
```

Comment:
if changing this line anyway, may as well change typo `levl` -> `level`

Reply:
fixed